aggregator/database/database.go

```diff
@@ -288,13 +288,19 @@ func (d *Database) FetchCheckpointMessages(fromTimestamp uint64, toTimestamp uin
 	operatorSetUpdateMessageAggregations := make([]messages.MessageBlsAggregation, 0, len(operatorSetUpdates))
 
 	for _, stateRootUpdate := range stateRootUpdates {
+		if stateRootUpdate.Aggregation == nil {
```

Comment:
I don't think we want to necessarily fail here. IMO we should just skip non-aggregated messages and log a warning for now.